tests/factories/QcResultUploadFactory.js

```diff
@@ -0,0 +1,27 @@
+import fs from 'fs'
+import BaseFactory from './BaseFactory.js'
+
+const defaultCsv = fs.readFileSync('./tests/data/csv/qc-results-upload.csv', 'utf8')
```

Comment:
Could we just put the file in here rather than having to load it synchronously?

That way we can get rid of the data file and use it in the e2e tests.

At the moment I do not think this is usable in the e2e tests as readFileSync is not available in the browser.

Reply:
Ok. I can see what you are saying.

I plan to look at this again as there are places we are using csvs when I think we could create them on the fly by using a temp file or something similar. e.g. pool creation.